### DISCUSSION TITLE
Can't seem to get my application approved

### DISCUSSION THREAD

**Discussion by billigproton** [community/community#120904] [GitHub Education]
### Hi everyone,

Hi everyone, I'm probably not the first one to have this issue, but I think I can't seem to find out what to do in my situation if I want to get my academic status approved. In the past, I got my academic status approved when I was enrolled at my previous school. Now, I put new academic email adress of my current school to my profile (BTW still didn't receive the verification email...) + I tried to upload my academic ID, but it showed the red box that the picture probably doesn't contain the ID of my current school (probably my old school still in the system). So, I can't get through the verification process and I honestly don't know what to do. Can anyone please help me? Thanks in advance!

**Comment by eliz-amst** [comment]
Have you added both emails to your account? Did you try uploading some other proof? Not the ID?

**Reply by billigproton** [reply]
Is it a problem when the proof is not in English?

**Reply by eliz-amst** [reply]
I believe you need to attach a translation from an agency or so, google translate won't get approved since it's not very trusty. I would try to get my documents translated at some agency and try to resubmit :) There is plenty agencies available online you can simply upload your documents and receive translation in 24 hours or so

**Comment by samus-aran** [comment]
Hi @billigproton 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

I suspect that the academic email you're using is associated with another school. If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this.
